tests/route/test_route_flap.py

```diff
@@ -94,40 +121,41 @@ def send_recv_ping_packet(ptfadapter, ptf_send_port, ptf_recv_ports, dst_mac, sr
 
 
 def get_ip_route_info(duthost):
-    output = json.loads(duthost.shell('vtysh -c "show ip route json"', verbose=False)['stdout'])
-    return output
+    output = json.loads(duthost.shell('vtysh -c "show ip bgp ipv4 json"', verbose=False)['stdout'])
+    return output['routes']
 
 
-def get_exabgp_port(tbinfo, nbrhosts):
-    tor_neighbors = natsorted([neighbor for neighbor in nbrhosts.keys()])
-    tor1 = tor_neighbors[0]
+def get_exabgp_port(duthost, tbinfo, dev_port):
+    tor1 = duthost.shell("show ip int | grep -w {} | awk '{{print $4}}'".format(dev_port))['stdout']
     tor1_offset = tbinfo['topo']['properties']['topology']['VMs'][tor1]['vm_offset']
     tor1_exabgp_port = EXABGP_BASE_PORT + tor1_offset
     return tor1_exabgp_port
 
 
-def test_route_flap(duthost, tbinfo, nbrhosts, ptfhost, ptfadapter, get_function_conpleteness_level):
+def test_route_flap(duthost, tbinfo, ptfhost, ptfadapter,
+                    get_function_conpleteness_level, announce_default_routes):
     ptf_ip = tbinfo['ptf_ip']
-    #dst mac = router mac
+    common_config = tbinfo['topo']['properties']['configuration_properties'].get('common', {})
+    nexthop = common_config.get('nhipv4', NHIPV4)
     dut_mac = duthost.facts['router_mac']
 
-    #get neighbor
-    mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
-    neighbor = mg_facts['minigraph_lo_interfaces'][0]['addr']
-
-    #get dst_prefix_list and nexthop
+    #get dst_prefix_list and aspath
+    routes = namedtuple('routes', ['route', 'aspath'])
     iproute_info = get_ip_route_info(duthost)
     dst_prefix_list = []
-    for route_prefix, route_info in iproute_info.items():
+    for route_prefix in iproute_info:
         if "/25" in route_prefix:
-            dst_prefix_list.append(route_prefix.strip('/25'))
-        for nexthops, nexthops_info in route_info[0].items():
-            if nexthops == 'nexthops':
-                for key, value in nexthops_info[0].items():
-                    if key == 'ip':
-                        nexthop = value
-                    if key == 'interfaceName':
-                        dev_port = value
+            # Use only multipath routes, othervise there will be announced new routes to T0 neigbours on t1 topo
+            multipath = iproute_info[route_prefix][0].get('multipath', False)
+            if multipath:
+                out = iproute_info[route_prefix][0].get('path').split(' ')
+                aspath = out[1:]
```

Comment:
hello, how do you decide aspath here?
e.g. u'path': u'65200 64931 65900'
aspath would be `[u'64931', u'65900']`, but why?